From 192e9481041cb925c6934f51b75f27aaf3421432 Mon Sep 17 00:00:00 2001
From: Kenneth Myhra <kennethmyhra@gmail.com>
Date: Thu, 12 Aug 2021 20:35:16 +0200
Subject: [PATCH 05/12] gio/ginetaddress.c: Serenity does not have
 IN_MULTICAST, just return 0

Since Serenity does not have IN_MULTICAST we just return 0
---
 gio/ginetaddress.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/gio/ginetaddress.c b/gio/ginetaddress.c
index ada32f8..7a0a0c2 100644
--- a/gio/ginetaddress.c
+++ b/gio/ginetaddress.c
@@ -746,8 +746,11 @@ g_inet_address_get_is_multicast (GInetAddress *address)
   if (address->priv->family == AF_INET)
     {
       guint32 addr4 = g_ntohl (address->priv->addr.ipv4.s_addr);
-
+#ifndef __serenity__
       return IN_MULTICAST (addr4);
+#else
+      return 0;
+#endif
     }
   else
 #ifdef HAVE_IPV6
-- 
2.25.1

